content/departments/technical-success/ce/process/license_keys.md

```diff
@@ -72,6 +72,20 @@ In most circumstances, license keys match the renewal dates on an account contra
 3. Follow the process below for generating a renewal license key. Unless otherwise noted during Step 1, use the same exact tags and user count as the existing key (make sure to comma separate each tag) and use the `"Active Contract Period End"` as the expiration date.
 4. Send the new license key to the customer (follow the process outlined below).
 
+## Revoking an active license
+
+In extreme circumstances, it may be required to revoke a customer's active Sourcegraph license. This will mark the license as invalid on Sourcegraph.com, and license verification checks from the customer's Sourcegraph instance will fail, ultimately disabling all Sourcegraph Enterprise features.
+
+To revoke an active license, follow these steps:
+
+1. Sign in to sourcegraph.com and visit the **Site-admin > Subscriptions** page.
+2. Navigate to the license that needs to be revoked.
+3. Click on the red **Revoke** button and provide a reason for the revocation.
+
+The next time the customer's instance performs a license verification check, the verification will fail and the customer's Enterprise features will be disabled.
+
+> Note: This will not work on licenses with a license that has the `allow-air-gapped` tag applied or the `enterprise-air-gap-0` license, as these license do not need to do any license verification checks.
```

Comment:
```suggestion
> Note: This will not work on special licenses with the `allow-air-gapped` tag or the `enterprise-air-gap-0` plan, as these license are not executing verification checks.
```